src/main/resources/META-INF/plexus/components.xml

```diff
@@ -9,6 +9,7 @@
 			</implementation>
 			<configuration>
 				<phases>
+					<clean>com.axonivy.ivy.ci:project-build-plugin:maven-dependency-cleanup</clean>

```

Comment:
does that work as expected? in theory I'd expect the clean lifecycle to be a distinct lifecycle ... can we bind mojos to it's phases anyway? See https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#lifecycle-reference

Reply:
![image](https://user-images.githubusercontent.com/42733123/106856103-4b6a2900-66be-11eb-8547-74f5577874e7.png)

Reply:
I don't see the command you entered here :)
The test should be: 
- run 'mvn clean' .... the default and our own mojo runs.
- run 'mvn verify'  (which essentially skips the complete 'clean' lifecycle) .... none of the 'clean' mojo runs.

Reply:
![image](https://user-images.githubusercontent.com/42733123/106861283-d8fd4700-66c5-11eb-84d0-1d55aded276a.png)
![image](https://user-images.githubusercontent.com/42733123/106861338-ec101700-66c5-11eb-9238-121e17d02259.png)